<launch>

  <arg name="robot_id" default="$(optenv ROBOT_ID unnamed_robot)"/>



  <group ns="$(arg robot_id)">

    <node pkg="sdpo_hangfaq2_driver" type="sdpo_hangfaq2_driver"
        name="sdpo_hangfaq2_driver" output="screen">

      <param name="encoder_res" value="48.0"/>
      <param name="gear_reduction" value="64.0"/>
      <param name="serial_port_name" value="/dev/ttyACM0"/>

    </node>

  </group>

</launch>
